scripts/utils.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+const fs = require("fs");
+const jsYaml = require("js-yaml");
+const moment = require("moment");
+const fetch = require("node-fetch");
```

Comment:
Could you please remove these dependencies from `deploy-component.js` where they are now unused? 😛

Reply:
I think I got rid of them now 😋